serverless.yml

```diff
@@ -4,7 +4,7 @@ app: photonranch
 service: photonranch-projects
 
 package:
-  exclude:
+  patterns:
```

Comment:
Noticed this in the calendar app too. Are you sure this keeps the same functionality? 
([serverless docs](https://www.serverless.com/framework/docs/providers/aws/guide/packaging#:~:text=Note%3A%20Don%27t%20forget%20to%20use%20the%20correct%20glob%20syntax%20if%20you%20want%20to%20exclude%20directories))

Reply:
You're right, I missed that - changed here and will change in the calendar repo as well. Thanks!